packages/web/utils/formatHelpers.ts

```diff
@@ -0,0 +1,33 @@
+export const convertToRoman = (_num: number): string => {
+  let num = _num;
+
+  type RomanOptions = {
+    [key: string]: number;
+  };
+
+  const roman: RomanOptions = {
+    M: 1000,
+    CM: 900,
+    D: 500,
+    CD: 400,
+    C: 100,
+    XC: 90,
+    L: 50,
+    XL: 40,
+    X: 10,
+    IX: 9,
+    V: 5,
+    IV: 4,
+    I: 1,
+  };
+  let str = '';
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const i of Object.keys(roman)) {
```

Comment:
`Object.keys(roman).forEach` or `.reduce` ?   Would not require the eslint comment